### PR TITLE
WRKR-1117

### PR DESCRIPTION
**Pull request by agnihotrigaurav**
JIRA-WRKR-1117 - output wercker logs in json format based on flag.

A flag "**--log-json**" has been added, In case this flag is present output of wercker internal logs statement would be in json format.

Testing - 

- Testing done locally, as flag has not been introduced for kiddie-pool.

- [WRKR-1117](https://jira.oraclecorp.com/jira/browse/WRKR-1117)  contains wrkr-1117_logs file that captures logs of testing.

- To switch output to json format run command "wercker --log-json build"

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -550,6 +552,9 @@ func GetApp() *cli.App {
 		if ctx.GlobalBool("debug") {
 			util.RootLogger().Formatter = &util.VerboseFormatter{}
 			util.RootLogger().SetLevel("debug")
+		} else if ctx.GlobalBool("log-json") {
```
[the remark this answers is not in the document]
Comment (agnihotrigaurav) [reply cmd/main.go]: Done

File: cmd/main.go [hunk 2]
```diff
@@ -550,6 +552,9 @@ func GetApp() *cli.App {
...
+			util.RootLogger().Formatter = &logrus.JSONFormatter{}
+			util.RootLogger().SetLevel("info")
```
[the remark this answers is not in the document]
Comment (agnihotrigaurav) [reply cmd/main.go]: Done

**Comment by agnihotrigaurav** [comment]
Updated PR with review comments. Kindly review again.

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ## unreleased
 
+- switch logs to json format(#535)
```
Comment (hatchan) [inline changelog.md]: `switch` -> `Switch`

**Pull request merged**